EpikCord/__init__.py

```diff
@@ -619,6 +619,118 @@ def heartbeater():
         except KeyError:
             return
 
+class Intents:
```

Comment:
Why did you add the Intents class again? Or did you delete it and move it up here?

Reply:
i delete and moved it up